data/csp/ec2/settings/micro/sle15/sp5/overlayfiles.yaml

```diff
@@ -0,0 +1,5 @@
+archive:
+  _namespace_ec2_cloud_init_cfg:
+    _include_overlays:
+      - cloud-cfg-ec2
+      - cloud-cfg-disable-network-config
```

Comment:
`cloud-cfg-disable-network-config` appears to be missing for Azure or is it inherited from somewhere in Azure? @jgleissner

Reply:
Seems I forgot to re-add the appropriate overlayfiles for Azure. Thanks for catching! I'll update the PR.